### PR TITLE
Test how gencodes are generated

### PR DESCRIPTION
**Pull request by rgrinberg**
I thought I saw a bug in there so wrote a little test case to prove (or
disprove). There's no bug but I think this is still a useful test case.
Especially given that this code can be further simplified.

### CODE REVIEW

**Comment by rgrinberg** [comment]
@AnthonyMl

File: src/main/java/org/protege/editor/owl/server/http/handlers/CodeGenHandler.java [hunk 1]
```diff
@@ -55,8 +55,31 @@ public void handleRequest(HttpServerExchange exchange) {
 		}
 	}
 
+	public static List<String> generateCodes(int seq, int cnt, String p,
```
Comment (bdionne) [inline src/main/java/org/protege/editor/owl/server/http/handlers/CodeGenHandler.java]: missing arguments here

Comment (rgrinberg) [reply src/main/java/org/protege/editor/owl/server/http/handlers/CodeGenHandler.java]: They're actually there but the alignment is way off: https://patch-diff.githubusercontent.com/raw/bdionne/protege-server/pull/3.diff

Let me see if I can fix it.

File: src/test/java/org/protege/editor/owl/server/http/handlers/CodeGenHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.when;
+
+public class CodeGenHandlerTest extends TestCase {
+
+    @Test
+    public void generateNTokens() {
+        List<String> codes = CodeGenHandler.generateCodes(10, 5, "p", "s", "_");
```
Comment (bdionne) [inline src/test/java/org/protege/editor/owl/server/http/handlers/CodeGenHandlerTest.java]: I'd add a test also for the garden path, which is a prefix "C", no suffix (null) and no delimiter (null)
